edk2toolext/invocables/edk2_ci_build.py

```diff
@@ -193,12 +244,8 @@ def Go(self):
         logging.log(edk2_logging.SECTION, "Loading plugins")
 
         pluginList = self.plugin_manager.GetPluginsOfClass(ICiBuildPlugin)
-        if len(self.packageList) == 0:
-            self.packageList.extend(self.PlatformSettings.GetPackages())
-
-        for pkgToRunOn in self.packageList:
 
-            # TODO: figure out if this package is a valid package
```

Comment:
We still need to figure out if this is a valid package or not

Reply:
This is handled in the Set function.  It raises exception if unknown or unsupported